extension/js/content_scripts/webmail/gmail_element_replacer.ts

```diff
@@ -82,22 +82,24 @@ export class GmailElementReplacer implements WebmailElementReplacer {
   setReplyBoxEditable = async () => {
     const replyContainerIframe = $('.reply_message_iframe_container > iframe').last();
     if (replyContainerIframe.length) {
-      $(replyContainerIframe).replaceWith(this.factory.embeddedReply(this.getReplyParams(this.getGonvoRootEl(replyContainerIframe[0])), true)); // xss-safe-value
+      $(replyContainerIframe).replaceWith(this.factory.embeddedReply(this.getLastMsgReplyParams(this.getGonvoRootEl(replyContainerIframe[0])), true)); // xss-safe-value
```

Comment:
Try this:
1. open a conversation where the first message was encrypted, and second was plain, and third was plain
2. make open the middle message, but not the first
3. click to reply to the middle message (plain) and wait for standard reply box to render
4. now click to open the first (encrypted) message in the conversation. element replacer should replace the reply box
5. I suspect it will replace the reply box thinking that you are replying to the last message in conversation? Even though user is replying to the middle one.

Reply:
The result:
![image](https://user-images.githubusercontent.com/49104848/68147219-4ad66600-ff42-11e9-9127-f0743a0970d4.png)

It replaces everything correctly but the standard reply box got replaced with `Draft Skipped`
I think would be better to leave it as a standard reply box or remove it. What do you think?

Reply:
@tomholub

Reply:
@michael-volynets I think it would be indeed appropriate to have it render a standard reply box, whatever Gmail would/should do, in that situation, or whatever you'd expect as user.